gui/wxpython/modules/histogram.py

```diff
@@ -258,7 +258,7 @@ def UpdateHistDone(self):
             return
         try:
             id = self.imagedict[self.img]
-        except:
+        except AttributeError:
```

Comment:
Why AttributeError?

Reply:
Incase either one of `self.imagedict` or `self.img` were Nonetype or not present

Reply:
Surely they are present looking couple lines above?? This should be KeyError.